fusion-plugin-i18n-react/src/with-translations.js

```diff
@@ -61,9 +61,8 @@ export const withTranslations = (
 
     const displayName = Component.displayName || Component.name || 'Anonymous';
     WithTranslations.displayName = `withTranslations(${displayName})`;
-    WithTranslations.contextTypes = {
-      i18n: PropTypes.object.isRequired,
-    };
+    // $FlowFixMe
+    WithTranslations.contextType = I18nContext;
```

Comment:
curious what was the issue here

Reply:
This was a surprise to me, too. I wasn't sure how to fix this
```
Cannot assign I18nContext to WithTranslations.contextType because property contextType is missing in statics of WithTranslations [1].
```